test/reply-error.test.js

```diff
@@ -335,14 +335,40 @@ test('invalid schema - ajv', t => {
   })
 })
 
+// Pull request 2884 https://github.com/fastify/fastify/pull/2884
```

Comment:
```suggestion
```

The comment is unnecessary. The `git blame` feature can be used to see when the test was added.

Reply:
Sorry, just following the kind of message I saw [elsewhere in the file](https://github.com/fastify/fastify/pull/2884/files/e071054e4b8bffd48a19773e84d070bfd3bd8ea5#diff-086d3e4983ab98bad387ce62fb47ff74c0646e94ef5caf4701d096160ea20c06R425). Will remove.